src/plugins/vis_type_timeseries/public/application/visualizations/lib/active_cursor.js

```diff
@@ -29,8 +29,23 @@
  */
 
 // TODO: Remove bus when action/triggers are available with LegacyPluginApi or metric is converted to Embeddable
-import $ from 'jquery';
 
```

Comment:
Since this file has changed completely, we should remove all of the copyright text and use this header:
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
Hi! Thanks, I fixed it!